Nonprofit-Exchange-Hub/web-app
# email verification

## Dev Summary

finally! sendgrid account with npc email and hooked it up to sign up endpoint


## Test Plan


repro steps:

1. get sendgrid api key from james
2. add FE_DOMAIN=http://localhost:3000 to the `.env` file
3. http://localhost:3000/signup
4. sign up with real email
5. try to sign in and fail
6. go to your email and verify
7. now sign in
8. success


## Resources
- https://docs.nestjs.com/fundamentals/testing#auto-mocking
